app/src/protyle/render/abcRender.ts

```diff
@@ -3,6 +3,30 @@ import {Constants} from "../../constants";
 import {genIconHTML} from "./util";
 import {hasClosestByClassName} from "../util/hasClosest";
 
+const ABCJS_PARAMS_KEY = "%%abcjsParams:";
+
+// Read the abcjsParams from the content if it exists.
+// The params *must* be the first line of the content in the form:
+// %%abcjsParams: {JSON}
+const getAbcParams = (abcString: string): any => {
+    let params = {
+        responsive: 'resize',
+    };
+
+    const firstLine = abcString.substring(0, abcString.indexOf("\n"));
+
+    if (firstLine.startsWith(ABCJS_PARAMS_KEY)) {
+        try {
+            const jsonString = firstLine.substring(ABCJS_PARAMS_KEY.length);
+            params = { ...params, ...JSON.parse(jsonString) };
```

Comment:
Maybe use this https://github.com/siyuan-note/siyuan/blob/449d2dbf87d838409351fc8c4b7a42af59dc65c3/app/src/util/functions.ts#L79

Reply:
Thanks. I've updated the code to use this.